roles/2-common/tasks/network.yml

```diff
@@ -24,3 +24,20 @@
     src: iptables
     dest: /etc/network/if-pre-up.d/iptables
     mode: '0755'
+
+# Ongoing rework (e.g. PR #2652) arising from ansible.posix collection changes:
+- name: Use 'sysctl' to set 5 network/kernel settings, turning off IPv6 if possible
```

Comment:
These are not all network related, and I have noted before IPV6 should only be disabled on br0